### ISSUE TITLE
Error after creating project using `npm init slidev@latest`

### ISSUE THREAD

**Issue by bram-pkg** [slidevjs/slidev#968]
**Describe the bug**
After creating a new project using `npm init slidev@latest`, the following occurs:

```
Failed to load url /node_modules/@slidev/client/constants (resolved id: /home/bram/src/cac-projects/slidev-test/node_modules/@slidev/client/constants) in /home/bram/src/cac-projects/slidev-test/node_modules/@slidev/client/App.vue. Does the file exist?
[vite:load-fallback] Could not load /home/bram/src/cac-projects/ee-project-overview-les-anciennes/node_modules/@slidev/client/constants (imported by node_modules/@slidev/client/App.vue?vue&type=script&setup=true&lang.ts): ENOENT: no such file or directory, open '/home/bram/src/cac-projects/ee-project-overview-les-anciennes/node_modules/@slidev/client/constants'
transforming (35) node_modules/@slidev/client/internals/Play.vue[Error: Could not load /home/bram/src/cac-projects/ee-project-overview-les-anciennes/node_modules/@slidev/client/constants (imported by node_modules/@slidev/client/App.vue?vue&type=script&setup=true&lang.ts): ENOENT: no such file or directory, open '/home/bram/src/cac-projects/ee-project-overview-les-anciennes/node_modules/@slidev/client/constants']
```

**To Reproduce**
1. Run `npm init slidev@latest`
2. Project name: slidev-test
3. Install and start it now: yes
4. Choose the agent: npm

**Desktop (please complete the following information):**
 - OS: Fedora 37
 - Browser: Chrome 112.0.5615.121
 - Slidev version: v0.40.13

**Comment by patrickdronk** [comment]
I have similar issues.

**Comment by KenN7** [comment]
same issue here

**Comment by 0xR** [comment]
Fixed it by running:
npm init slidev@0.29.2

no luck with:
0.39.0
0.40.12
0.40.13 (latest)

**Comment by tonai** [comment]
I have an "old" version that is working with slidev 0.40.5
If I remove package-lock and npm install then it's not working anymore.
It might be related to some dependencies and not related to slidev itself...

**Comment by lihbr** [comment]
Temporary fix for users, you can explicitly declare Vite as a dependency of the project to force its resolution to an older version:
```json
"devDependencies": {
	"vite": "2.2.2"
}
```

**Comment by antfu** [comment]
Fixed with `^0.40.14`
